images/scripts/update-cilium-envoy-image.sh

```diff
@@ -30,17 +34,18 @@ fi
 
 echo "Latest image from branch ${github_branch}: ${image_full}"
 
-echo "Updating image in ./images/cilium/Dockerfile"
-sed -i -E "s|(FROM ${image}:)(.*)(@sha256:[0-9a-z]*)( as cilium-envoy)|\1${image_tag}@${image_sha256}\4|" ./images/cilium/Dockerfile
+DOCKERFILEPATH="./images/cilium/Dockerfile"
+echo "Updating image in ${DOCKERFILEPATH}"
+sed -i -E "s|(FROM ${image}:)(.*)(@sha256:[0-9a-z]*)( as cilium-envoy)|\1${image_tag}@${image_sha256}\4|" ${DOCKERFILEPATH}
 
-echo "Updating image in ./install/kubernetes/cilium/values.yaml.tmpl"
-# Using tr to workaround matching the multiline regex with sed
-# yq would change formatting: https://github.com/mikefarah/yq/issues/465
-# use of envoy.image.override (which would allow match in one line) isn't optimal either
-< ./install/kubernetes/cilium/values.yaml.tmpl tr '\n' '\f' |
-  sed -E "s|(# -- Envoy container image\..*tag: \")(v[0-9a-zA-Z\.-]*)(\")|\1${image_tag}\3|" |
-  sed -E "s|(# -- Envoy container image\..*digest: \")(sha256:[0-9a-z]*)(\")|\1${image_sha256}\3|" |
-  tr '\f' '\n' > ./install/kubernetes/cilium/values.yaml.tmpl_tmp &&
-  mv ./install/kubernetes/cilium/values.yaml.tmpl_tmp ./install/kubernetes/cilium/values.yaml.tmpl
+MAKEFILEPATH="./install/kubernetes/Makefile.values"
+echo "Updating image in ${MAKEFILEPATH}"
+sed -i -E "s|export[[:space:]]+CILIUM_ENVOY_VERSION:=.*|export CILIUM_ENVOY_VERSION:=${image_tag}|"   ${MAKEFILEPATH}
```

Comment:
looks like there are some unnecessary whitespaces in front of the `${MAKEFILEPATH}` (on both lines)